File: 백준/Silver/14582. 오늘도 졌다/오늘도 졌다.py
a=list(map(int, input().split()))
b=list(map(int, input().split()))
team_a=0
team_b=0
flag=0
for i in range(len(a)):
    team_a += a[i]
    if team_a > team_b :
        flag=1
    team_b += b[i]
if flag==1 and team_b > team_a :
    print('Yes')
else :
    print('No')

File: 백준/Silver/14582. 오늘도 졌다/오늘도 졌다.cc
#include <iostream>
using namespace std;

int main() {
    int j[10], s[10], flag = 0, total_j = 0, total_s = 0;

    for (int i = 1; i <= 9; i++) cin >> j[i];
    for (int i = 1; i <= 9; i++) cin >> s[i];

    for (int i = 1; i <= 9; i++) {
        total_j += j[i];
        if (total_j > total_s)
            flag = 1;
        total_s += s[i];
    }

    if (flag == 1 && total_s > total_j)
        cout << "Yes";
    else
        cout << "No";

    return 0;
}